`caniusepython3` don't see taxjar library

Here is what I see in console output.
```
>>> caniusepython3 -r requirements.txt

Finding and checking dependencies ...
[ERROR] Failed to get external data for https://www.red-dove.com/pypi/projects/T/taxjar/project.json: HTTP Error 404: Not Found
Traceback (most recent call last):
  File "~/venv3/lib/python3.6/site-packages/distlib/util.py", line 913, in _get_external_data
    resp = urlopen(url)
  File "/usr/lib/python3.6/urllib/request.py", line 223, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.6/urllib/request.py", line 532, in open
    response = meth(req, response)
  File "/usr/lib/python3.6/urllib/request.py", line 642, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.6/urllib/request.py", line 570, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.6/urllib/request.py", line 504, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.6/urllib/request.py", line 650, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found
[WARNING] taxjar not found
```

I know, that in documentation you shown that python 3+ is supporting but still this may help.

--- a/setup.py
+++ b/setup.py
@@ -9,6 +9,20 @@ setup(
     url='https://github.com/taxjar/taxjar-python',
     download_url='https://github.com/taxjar/taxjar-python/archive/v1.7.0.zip',
     packages=['taxjar', 'taxjar.data'],
+    classifiers=[
+        "Programming Language :: Python :: 2.6",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
+        "Development Status :: 5 - Production/Stable",
+        "Intended Audience :: Developers",
+        "Natural Language :: English",
+        "Topic :: Office/Business",
+        "Topic :: Office/Business :: Financial",
+        "Topic :: Office/Business :: Financial :: Accounting",
+        "Topic :: Software Development :: Libraries :: Python Modules"
+    ],
     install_requires=[
         'requests >= 2.13.0',
         'jsonobject == 0.7.1'

